src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml.cs

```diff
@@ -67,50 +60,52 @@ public void Refresh()
 
         private void ProjectInstallButtonClicked(object sender, EventArgs e)
         {
-            var model = (DetailControlModel)DataContext;
+            var model = (PackageDetailControlModel)DataContext;
```

Comment:
this would never throw, since it will always be able to typecast. were you able to repro this exception for install button?

Reply:
This does not throw. Since `DataConext` was null, `model` was also null and then trying to access `model.SelectedVersion` would throw.

Reply:
ohh yes, this will be specific to project level.